pkg/cluster/k8sres.go

```diff
@@ -893,10 +895,20 @@ func (c *Cluster) getPodEnvironmentSecretVariables() ([]v1.EnvVar, error) {
 		return secretPodEnvVarsList, nil
 	}
 
-	secret, err := c.KubeClient.Secrets(c.Namespace).Get(
-		context.TODO(),
-		c.OpConfig.PodEnvironmentSecret,
-		metav1.GetOptions{})
+	secret := &v1.Secret{}
+	err := retryutil.Retry(10*time.Second, 3*time.Minute,
```

Comment:
@snarlysodboxer what do you thinks about to don't hardcode these values and utilize configurable values from the `Resources.ResourceCheckInterval` and `Resources.ResourceCheckTimeout`

Reply:
I was thinking about this as well. The values here are not too far off from the defaults of `ResourceCheckInterval` and `ResourceCheckTimeout` so I would vote to use them @snarlysodboxer .